lib/common/graph.js

```diff
@@ -219,8 +219,13 @@ class ConcertoGraphVisitor extends DiagramVisitor {
             if(argument && argument.name) {
                 const fqn = modelFile.getFullyQualifiedTypeName(argument.name);
                 if (fqn) {
-                    parameters.graph.addVertex(fqn);
```

Comment:
Is this line redundant?

Reply:
Yes it is, because we don't need a vertex for the decorator as it now lives within the umbrella of the parent(modelfile/declaration)

Reply:
Adding this back as a no-op to guarantee a vertex always exists for the decorator type reference